README.md

```diff
@@ -4,6 +4,8 @@
 
 [![build status](http://travis-ci.org/cc65/cc65.png)](http://travis-ci.org/cc65/cc65/builds)
```

Comment:
That line is strange.  It is different in cc65:master (see dd75d3f8841b5828e173de5fe7f4a7fc8dfe7c47); but, it isn't flagged as a change here.  It might cause a merge conflict.